src/features/root/MainSection.tsx

```diff
@@ -23,28 +21,8 @@ export default function MainSection({ content }: MainSectionProps) {
         return <MainSectionSkeleton />
     }
 
-    const renderModules = (modulesGroups: ModuleContent[][]) => {
-        return modulesGroups.map((modules, groupIndex) => (
-            <QuickLinkContainer key={groupIndex}>
-                {modules.map((module, index) => (
-                    <QuickLinkLink
-                        key={index}
-                        quickLink={{
-                            href: module.url,
-                            icon: getModuleEmoji(module.code),
-                            id: index,
-                            name: module.code,
-                        }}
-                    />
-                ))}
-
-                <div className="my-2 w-full"></div>
-            </QuickLinkContainer>
-        ))
-    }
-
     return (
-        <div className="space-y-8">
+        <div id='main-content' className="space-y-8">
```

Comment:
skip to main content still doesn't work

maybe create `<main>` with that id a bit lower with actual main content